packages/block-editor/src/components/segmented-text-control/index.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { BaseControl, Button } from '@wordpress/components';
+
+/**
+ * @typedef {Object} Control
+ * @property {string} label The label of the control.
+ * @property {string} value The value of the control.
+ * @property {string} icon  The icon of the control.
+ */
+
+/**
+ * Control to facilitate segmented text selections.
```

Comment:
```suggestion
 * Control to facilitate selecting a text style from a set of options.
```
To me, this comment doesn't seem to match what this component does.